spec/views/user/_user_listing_single.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+# -*- encoding : utf-8 -*-
+require 'spec_helper'
+
+describe 'when displaying user listings' do
+  let(:highlighted_words) { [] }
+  let(:user){ FactoryGirl.create(:user) }
+
+  before do
+    assign :highlight_words, highlighted_words
+  end
+
+  def render_view
+    render :partial => 'user/user_listing_single',
+           :locals => { :display_user => user }
+  end
+
+  it 'displays a normal request' do
+    info_request = FactoryGirl.create(:info_request, :user => user)
```

Comment:
Don't really need to assign these as they're unused – initially added a little confusion of "where are these being used?"